.eslintrc

```diff
@@ -0,0 +1,25 @@
+{
+  "extends": [
+    "standard",
+    "prettier",
+    "prettier/standard"
+  ],
+  "plugins": [
+    "import",
+    "prettier",
+    "standard"
+  ],
+  "parserOptions": {
+    "ecmaVersion": 2017 
+  },
+  "env": {
+    "node": true,
+    "es6": true,
+    "jest": true
+  },
+  "rules": {
+    "space-before-function-paren": 0,
```

Comment:
https://www.npmjs.com/package/eslint-plugin-react-hooks
Might want to add this and a plugin for it both made by React team